tests/test_wrapper.py

```diff
@@ -4,7 +4,8 @@
 
 
 def test_translate():
-    assert DnaSequence("AAAGGGAAA").translate(table=1) == ProteinSequence("KGK")
+    assert DnaSequence("AAAGGGAAA").translate(
+        table=1) == ProteinSequence("KGK")
```

Comment:
Weird formatting here?

Reply:
note to self: also add python formatter and linter to this project. Given that there is some amount of python code. So far we only have Rust.

Reply:
This is actually the result of my Python formatter (autopep8 I think). I don't like it either though.

Reply:
we use `black`, maybe it outputs slightly different formats

Reply:
https://github.com/SecureDNA/quickdna/pull/23